docs/guide-install.md

```diff
@@ -4,33 +4,35 @@ category: getting-started
 order: 10
 ---
 
-You can install abapGit with an online install or an offline install.
+## Summary ##
+abapGit exists in 2 flavours: _standalone_ version or _developper_ version. 
 
-Offline install steps are: 
-1. Create ABAP report installer
-2. Run installer ABAP report to perform offline projet import
+  * The standalone version is targeted at users. It consist of one (huge) program which contains all the needed code. You run the standalone version in transaction `SE38`, executing the program you created.
+  * The developer version is targeted at developers contributing to the abapGit codebase. It consists of all the ABAP programs/classes/interfaces/etc. of the abapGit project. You run the developer version with transaction `ZABAPGIT`.
 
-Online install steps are: 
-1. Create ABAP report installer
-2. Setup SSL
-3. Run installer ABAP report to perform online projet import
+## Prerequisites ##
+abapGit requires SAP BASIS version 702 or higher.
 
-## Create ABAP Report installer ##
+## Install standalone version ##
 1. Download the [ABAP code](https://raw.githubusercontent.com/abapGit/build/master/zabapgit.abap)(right click -> save-as) to a file. 
-2. Via SE38 or SE80, create a new report named `ZABAPGIT_FULL`.
-3. Upload the code from the file using Utilities -> More Utilities -> Upload/Download -> Upload
+2. Via `SE38` or `SE80`, create a new report named `ZABAPGIT_STANDALONE` (formerly `ZABAPGIT_FULL`). NB: Don't use the name `ZABAPGIT` if you plan to install the developer version.
+3. In source code change mode, upload the code from the file using Utilities -> More Utilities -> Upload/Download -> Upload
 4. Activate
 
-Typically abapGit will only be used in the development system, so it can be installed in a local `$` package (eg: `$ZABAPGIT`).
+Typically abapGit will only be used in the development system, so it can be installed in a local `$` package (eg: `$TMP` or `$ZABAPGIT`).
```

Comment:
```suggestion
Typically, abapGit will only be used in the development system, so it can be installed in a local `$` package (e.g.  `$ZABAPGIT`).
```